acquia/acquia_cms
# ACMS-915: Address compatibility with Drupal's Standard installation profile in Drupal 9.3.x

**Motivation**

Fixes #[ACMS-915](https://backlog.acquia.com/browse/ACMS-915)

**Proposed changes**

Move config in optional directory as its already provided by standard profile.

**Alternatives considered**

NA
**Testing steps**


- Create drupal project with 9.3.x-dev using below command
```
composer create-project drupal/recommended-project:9.3.x-dev drupal9-dev
```
- Install site using standard profile
```
drush si account-pass admin
```
- Add `acquia_cms_article` module
```
composer require drupal/acquia_cms_article
```
- Enable `acquia_cms_article` module
```
drush en acquia_cms_article
```
- Make sure you don't see any error during site install
- Make sure user role Content editor is created

**Merge requirements**
